src/styles/headings/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Headings
+description: Use headings to structure content on pages
+section: Styles
+theme: Typography
+layout: layout-pane.njk
+showPageNav: true
+order: 7
+---
+
+{% from "_example.njk" import example %}
+
+Write all headings in sentence case.
```

Comment:
This line feels like it should go further down the page. The following paragraph (currently line 15) feels a bit more natural as an opening paragraph for this page.

Reply:
Decision – move to 3rd para